tests/Feature/UserRoleMiddlewareTest.php

```diff
@@ -1,135 +0,0 @@
-<?php
-
-namespace Tests\Feature;
-
-use App\Models\User;
-use Illuminate\Foundation\Testing\RefreshDatabase;
-use Route;
-use Tests\TestCase;
-
-class UserRoleMiddlewareTest extends TestCase
```

Comment:
Please add Tests for the admin backend's accessiblity.

Otherwise I'm fine w/ everything else